pkg/audit/timeout_test.go

```diff
@@ -77,8 +78,12 @@ func TestGetShareTimeout(t *testing.T) {
 			require.NoError(t, err)
 		}
 
-		_, err = verifier.Verify(ctx, stripe)
-		require.NoError(t, err)
+		verifiedNodes, err := verifier.Verify(ctx, stripe)
+		assert.Error(t, err)
```

Comment:
requires over asserts now right?

Reply:
No. Here using assert over requires is intentional. I want to check both `err` and `verifiedNodes`. If the test fails here, this will give us more clue what went wrong.